src/databases/config/config.ts

```diff
@@ -0,0 +1,34 @@
+import dotenv from 'dotenv'
+dotenv.config()
+

```

Comment:
Database for Development Environment should be different from Production Environment. So, Let's use the local database in the Development Environment and then use the Hosted Database in the Production Environment.

Reply:
Done